__tests__/validators/stale.test.js

```diff
@@ -94,6 +96,11 @@ const createMockContext = (results) => {
     })
   }
 
-  context.github.issues.createComment = jest.fn()
+  // context.github.search = {
```

Comment:
please remove un need codes

Reply:
Done! Remnants of debugging code. Thanks for pointing it out. :-|